rom/dev/README.md

```diff
@@ -687,3 +687,6 @@ The following are the pre-conditions that should be satisfied:
         - The TOC hash will NOT match, skip the TOC hash validation.
         - We still need to make sure that the hash of the FMC which was stored in the data vault register at cold boot
           still matches the FMC code. This is the hash of the FMC image portion.
+    - If validation fails during ROM boot, the new image will not be copied from
```

Comment:
This is an interesting tradeoff. I suppose FMC can take a different policy choice if we change our minds here. @vsonims to note this.

Reply:
The main motivation for this is to prioritize availability, which I think aligns with priorities I've heard in the past. Also I think it aligns with the spirit of Impactless Update.